client/src/www/app/outline_server_repository/config.ts

```diff
@@ -109,82 +77,44 @@ export function setTransportConfigHost(
  * This is used by the server to parse the config fetched from the dynamic key, and to parse
  * static keys as tunnel configs (which may be present in the dynamic config).
  */
-export function parseTunnelConfig(
+export async function parseTunnelConfig(
```

Comment:
The user-provided dynamic key error is removed here, are we still supporting this feature? You can use the following key to test, expect to see "Usage quota exceeded 😭😭😭":

```
https://raw.githubusercontent.com/jyyi1/jyyibin/refs/heads/main/keys/error_msg.okey#Dynamic%20Error%20Msg
```

Reply:
Good catch. https://github.com/Jigsaw-Code/outline-apps/pull/2329/files will have a fix